.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ ] Great on set up the linters files but I think to install the packages for hintrc and stylelint if you scroll down [here](https://github.com/microverseinc/linters-config/tree/master/html-css) you will see the steps please add it to your project ✅ and please don't forget after installing the packages to put `node_modules` in gitignore file 🚀🚀
![image](https://user-images.githubusercontent.com/58553711/184869109-2fd349fa-063c-4d4e-a5a1-72ebf482bc4c.png)

Reply:
Thank you for reviewing the project
I set up all requirements I think If I merge everything is going come together
<img width="126" alt="image" src="https://user-images.githubusercontent.com/88346506/184900688-01f43dce-8f85-4542-b24c-74a599700e34.png">